mods/persistence/modules/science/files/programs/bluprintr.dm

```diff
@@ -114,16 +114,16 @@
 			to_chat(user, SPAN_NOTICE("Unable to find experiment [href_list["PRG_beginexperiment"]]."))
 			return
 		var/datum/fabricator_recipe/experimental_device/ED = new(experiment.id, experiment.filename)
-		if(!protolathe.try_queue_build(ED, 1))
-			to_chat(user, SPAN_WARNING("Unable to start experiment. A technical error has occured with the protolathe."))
-			QDEL_NULL(ED)
-			return
+		protolathe.try_queue_build(ED, 1)
```

Comment:
Why was this removed?

Reply:
try_queue_build() doesn't actually return a value, so this check always failed. Overriding the try_queue_build proc a second time for the sake of this one instance seemed unnecessary to me, but if you think it is, then I can readd the check in some way.